src/scripts/main.js

```diff
@@ -2,7 +2,67 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+function dashboardGenerator(list) {
+  for (const value of list) {
+    const newRow = document.createElement('tr');
+
+    value.age = value.died - value.born;
+    value.century = Math.ceil(value.died / 100);
+
+    for (const key in value) {
+      const rowData = document.createElement('td');
+
+      switch (key) {
+        case 'name':
+          rowData.textContent = value[key];
+          newRow.append(rowData);
+          break;
+
+        case 'sex':
+          switch (value[key]) {
+            case 'f':
+              rowData.textContent = 'female';
+              break;
+
+            case 'm':
+              rowData.textContent = 'male';
```

Comment:
```suggestion
              rowData.textContent = 'Male';
```
it is reason of failed tests